lib/context.js

```diff
@@ -273,7 +274,12 @@ class ContextSession {
    */
 
   async remove() {
-    const opts = this.opts;
+    // Override the default options so that we can properly expire the session cookies
+    const opts = Object.assign({}, this.opts, {
+      expires: new Date(COOKIE_EXP_DATE),
```

Comment:
`const COOKIE_EXP_DATE = new Date('Thu, 01 Jan 1970 00:00:00 GMT');`

so we don't need to create a date instance every time.